### PR TITLE
Enhance Utilities Documentation Writing

### PR DESCRIPTION
**Pull request by SHi-ON**
## Linked Issue

Closes #{issueNumber}

## Description

This PR covers a couple of corrections of writing mistakes and a few minor enhancements in order to enhance the documentation quality. Please note that the scope of this PR is the components under `utilities` only.

## Changsets

Instructions: Changesets automate our changelog. If you modify files in `/packages`, run `pnpm changeset` in the root of the monorepo, follow the prompts, then commit the markdown file. Changes that add features should be `minor` while chores and bugfixes should be `patch`. Please prefix the changeset message with `feat:`, `bugfix:` or `chore:`.

## Checklist

Please read and apply all [contribution requirements](https://www.skeleton.dev/docs/contributing).

- [x] This PR targets the `dev` branch (NEVER `master`)
- [x] Documentation reflects all relevant changes
- [x] Branch is prefixed with: `docs/`, `feat/`, `chore/`, `bugfix/`
- [x] Ensure Svelte and Typescript linting is current - run `pnpm ci:check`
- [x] Ensure Prettier linting is current - run `pnpm format`
- [x] All test cases are passing - run `pnpm test`
- [x] Includes a changeset (if relevant; see above)

### CODE REVIEW

**Comment by endigo9740** [comment]
Thank you @SHi-ON. Note we do typically request folks give us a heads up before submitting changes. This helps avoid redundant effort. Additionally I would likely suggested we submit each doc change as a dedicated PR. Sweeping changes across lots of unrelated sections can sometimes be detrimental.

That said, this looks fine so I'd expect these to be merge soon. I typically do PR reviews over the weekend now, so expect this soon.

**Comment by SHi-ON** [comment]
Thanks for the heads-up @endigo974, I'll keep them in mind next time.

File: sites/skeleton.dev/src/routes/(inner)/utilities/popups/+page.svelte [hunk 1]
```diff
@@ -578,7 +578,7 @@ const popupState: PopupSettings = {
 		<section class="space-y-4">
 			<h2 class="h2">Combobox</h2>
 			<p>
-				The goal of Skeleton is to combine primitive elements and components to build more complex UI. For example, by combining a Button,
+				The goal of Skeleton is to combine primitive elements and components to build a more complex UI. For example, by combining a Button,
```
Comment (phamduylong) [inline sites/skeleton.dev/src/routes/(inner)/utilities/popups/+page.svelte:581]: I'd make the argument that *to build more complex user interfaces/UIs* sounds better here. It just sounds weird saying you're making "a UI", at least to me. Not a big deal tho.

Comment (SHi-ON) [reply sites/skeleton.dev/src/routes/(inner)/utilities/popups/+page.svelte:581]: My correction was solely following the English grammar. It makes sense to me as in "a cross-platform UI".

Comment (endigo9740) [reply sites/skeleton.dev/src/routes/(inner)/utilities/popups/+page.svelte:581]: Don't stress about it guys. I'll review and make adjustments as needed. Thanks for the help and input to both of you though!

**Comment by endigo9740** [comment]
@SHi-ON @phamduylong thank you for your your help. Note that I've follow up with a few additions.

- We use the same line for the modals/drawers/toasts to inform users they need to read the full page documentation. This is now uniform between all three pages.
- Some context was not quite right in a couple of the suggested changes. In these cases I've rewrote the line to make the intention more clear. Most of this was subtle edge cases where an English term can be used to have different meanings (ex: proceed)

Overall these are great changes. Thanks for catching the typos especially!

**Pull request merged**